---
title: C28163
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C28163
helpviewer_keywords:
- C28163
ms.assetid: 24fecbde-1c96-4a45-82f7-9f47cfc0ef11
author: mikeblome
ms.author: mblome
manager: wpickett
ms.workload:
- multiple
ms.openlocfilehash: d820ee4244c03368cefc5b9b4af4aa9df11ffdb7
ms.sourcegitcommit: 94b3a052fb1229c7e7f8804b09c1d403385c7630
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 04/23/2019
ms.locfileid: "62574763"
---
# <a name="c28163"></a>C28163
警告 C28163:该函数应永远不会调用从在一个 try / except 块

 当函数的类型时，将报告此警告，永远不应括在`try/except`中找到该块`try/except`块。 代码分析工具至少发现了一个路径，其中，所调用的函数都位于 `try/except` 块中。